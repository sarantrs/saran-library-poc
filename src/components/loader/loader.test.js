import React from "react";
import TextInput from "./loader";
import renderer from "react-test-renderer";
import Loader from "./loader";

describe("TextInput", () => {
  it("renders properly", () => {
    const tree = renderer
      .create(<Loader />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
