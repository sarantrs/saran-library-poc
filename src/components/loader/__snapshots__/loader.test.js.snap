// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInput renders properly 1`] = `
<div
  className="loader-wrapper-center "
>
  <div
    className="los-custom-loading"
  >
    <div
      className="loader"
    />
    <div
      dangerouslySetInnerHTML={
        Object {
          "__html": "Loading...",
        }
      }
    />
  </div>
</div>
`;
